xgboost_ames_housing/Dockerfile

```diff
@@ -21,6 +21,9 @@ FROM python:2.7
 RUN apt-get update
 RUN apt-get install -y git make g++
 
+# Download the training set to local directory 'ames_dataset'.
+COPY ames_dataset/ ./ames_dataset
```

Comment:
Do we also need to copy the python code to the container? Seems kind of weird that we need to download the actual python script from Github.

Reply:
Good point. Change it to copy the code needed only from local files.